### PR TITLE
Add bibliography for chapter 10 and link citations. Closes #116

### LINKED ISSUE #116: Chapter 10 Citation To-Dos
[Link](https://bookdown.org/yihui/bookdown/citations.html) to Bookdown citations documentation.
- [x] Read through chapter text, making sure that each citation is present and properly formatted.
- [x] Add citations from book.bib file to a new collection for the chapter in Zotero.
- [x] Add any missing citations manually into Zotero.
- [x] Create new branch for the chapter.
- [x] Export the chapter's .bib file as BibTeX and save into the bibliographies folder within textbook.
- [x] Add the .bib file into the array labeled "bibliography" within the index.Rmd file.
- [x] Link each inline citation to its .bib listing equivalent.
- [x] Make sure that each inline citation links to the corresponding reference within the bibliography section of the chapter.

### PR DESCRIPTION
**Pull request by nickmmurphy**

### CODE REVIEW

File: bibliographies/ch10.bib [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+	number = {1},
+	journal = {Mathematical Geology},
+	author = {Goovaerts, Pierre},
+	year = {2008},
+	pmid = {1000000221},
+	note = {ISBN: 6176321972
+\_eprint: NIHMS150003},
```
Comment (ect123) [inline bibliographies/ch10.bib:250]: Looks like there's a missing `},` on the line above, and I'm not sure if the `\_` is intentional.

File: bibliographies/ch10.bib [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+	author = {Cressie, N. A. C.},
+	year = {1994},
+	doi = {10.2307/2533238},
+	note = {ISSN: 0006341X
+Issue: 1
```
Comment (ect123) [inline bibliographies/ch10.bib:295]: I'm not sure if this is another case of the missing `},` on the line above

File: bibliographies/ch10.bib [hunk 3]
```diff
@@ -0,0 +1,481 @@
...
+	title = {An {Introduction} to {Applied} {Geostatistics}.},
+	volume = {17},
+	url = {https://www.sciencedirect.com/science/article/pii/009830049190055I},
+	number = {3},
+	publisher = {Oxford University Press, New York},
+	author = {Isaaks, Edward and Srivastava, R.Mohan},
+	year = {1989},
+	doi = {https://doi.org/10.1016/0098-3004(91)90055-I},
+	note = {ISSN: 0098-3004
+Publication Title: Oxford University Press},
```
Comment (ect123) [inline bibliographies/ch10.bib:325]: missing `},` on the line above?

**Review by ect123 — CHANGES_REQUESTED** [review]
Looks like there might have been an issue with the BibTex export that didn't add an `},` at the end of a few elements of some of the records. Could you take a look and check to see if this needs to be edited?

File: bibliographies/ch10.bib (hunk 1 above)
Comment (nickmmurphy) [reply bibliographies/ch10.bib:250]: Looks like this is a Zotero artifact. The page it scraped from must have added additional information into the "notes" section, so this is actually one line (and it should work), but it's not necessary so I removed it!.

**Comment by nickmmurphy** [comment]
> Looks like there might have been an issue with the BibTex export that didn't add an `},` at the end of a few elements of some of the records. Could you take a look and check to see if this needs to be edited?

Hi Evan, these are actually just multi-line elements. The Zotero entries were imported from the old book.bib, and it looks like there was a bunch of information put into the "Notes" sections. None of these are going to affect how the bibliographies are displayed, but I removed them anyway and pushed the change.

**Review by ect123 — APPROVED** [review]

**Pull request merged**